tests/check/check_ttu.go

```diff
@@ -0,0 +1,181 @@
+package check
+
+import (
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"google.golang.org/protobuf/types/known/structpb"
+
+	checktest "github.com/openfga/openfga/internal/test/check"
+)
+
+var ttuCompleteTestingModelTest = []*stage{
+	{
+		Name: "direct_pa_direct_ch",
+		Tuples: []*openfgav1.TupleKey{
+			{Object: "directs-user:ttu_1", Relation: "direct", User: "user:valid"},
+			{Object: "ttus:1", Relation: "mult_parent_types", User: "directs-user:ttu_1"},
+			{Object: "directs-user:ttu_1_invalid", Relation: "direct", User: "user:ttu_invalid_group"},
+		},
+		CheckAssertions: []*checktest.Assertion{
+			{
+				Name:        "valid_user",
+				Tuple:       &openfgav1.TupleKey{Object: "ttus:1", Relation: "direct_pa_direct_ch", User: "user:valid"},
+				Expectation: true,
+			},
+			{
+				Name:        "invalid_user",
+				Tuple:       &openfgav1.TupleKey{Object: "ttus:1", Relation: "direct_pa_direct_ch", User: "user:invalid"},
+				Expectation: false,
+			},
+			{
+				Name:        "invalid_group",
+				Tuple:       &openfgav1.TupleKey{Object: "ttus:1", Relation: "direct_pa_direct_ch", User: "user:invalid_group"},
+				Expectation: false,
+			},
```

Comment:
Nitpick: unsure what value this test is adding, it looks identical to the one above it

Reply:
Will change to ttu_invalid_group.  Want to test the case where there is such user belong to `directs-user`, but that `directs-user` does not connect to the TTU itself.